src/Library/demos/Text View/main.vala

```diff
@@ -0,0 +1,76 @@
+    -1
+  );
+
+  copy.clicked.connect(handleCopy);
+  paste.clicked.connect(handlePaste);
+  cut.clicked.connect(handleCut);
+  select.clicked.connect(() => {
+    Gtk.TextIter start, end;
+
+    buffer.get_start_iter(out start);
+    buffer.get_end_iter(out end);
+
+    buffer.select_range(start, end);
+  });
+  clear.clicked.connect(() => {
+    Gtk.TextIter start, end;
+
+    buffer.get_start_iter(out start);
+    buffer.get_end_iter(out end);
+
+    buffer.@delete(ref start, ref end);
+  });
+
+  textview.copy_clipboard.connect(() => {
```

Comment:
If there is only one function call the `{ }` can be left out:
`textview.copy_clipboard.connect(() => message (""));`